canonical-web-and-design/ubuntu.com
# Build engage page and takeover - OpenStack deployment guide (es)

## Done

- Build engage page
- Build takeover
- Add them to the 3 pages
- Build social banner

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/es/guía-implementación-openstack and check content matches [copy doc](https://docs.google.com/document/d/1nyF_VA2b5K-Un_Ty9LJBo3Ph4taMlUxv7RdoN4W6ngk/edit) and [brief](https://docs.google.com/spreadsheets/d/16J2T4D9VW0ZUyfBMht_G_z7Ia0mnHKIMcyV_22s4WJk/edit?ts=5f479112#gid=1271849439)
- Change browser primary language to Spanish and check takeover. 
- Check [social banner](https://tools.demos.haus/?title=Gu%C3%ADa+para+la+implementaci%C3%B3n+de+OpenStack&subtitle=Superar+los+desaf%C3%ADos+de+la+adopci%C3%B3n+de+OpenStack+siguiendo+un+proceso+de+adopci%C3%B3n+probado&illustration_url=https%3A%2F%2Fassets.ubuntu.com%2Fv1%2Fdcb2963c-openstack%2Bcloud%2Boutlines.svg&background=dark#banner-form-modal)
- Test the engage page on [http://debug.iframely.com/](http://debug.iframely.com/) and [https://cards-dev.twitter.com/validator](https://cards-dev.twitter.com/validator)


## Issue / Card

Fixes #3175
